f5/bigip/resource.py

```diff
@@ -568,6 +568,30 @@ def _produce_instance(self, response):
         new_instance._activate_URI(new_instance.selfLink)
         return new_instance
 
+    def _reduce_boolean_pair(self, config_dict, key1, key2):
```

Comment:
@zancas: Moved this method up to ResourceBase because the class Failover is an UnnamedResource, which inherits from ResourceBase, not resource.